planner/core/plan_cost.go

```diff
@@ -253,6 +258,13 @@ func (p *PhysicalIndexLookUpReader) estDoubleReadCost(tbl *model.TableInfo, cost
 	return (numDoubleReadTasks * seekFactor) / concurrency
 }
 
+// GetNetworkCost calculates the cost of the plan in network data transfer.
+func (p *PhysicalIndexLookUpReader) GetNetworkCost() float64 {
```

Comment:
Actually the returned value is not a cost. How about renaming this function to `GetNetDataSize` or `GetNetDataBytes`?

Reply:
ok